kernel/packages/entryPoints/unity.ts

```diff
@@ -39,6 +40,16 @@ const observer = renderStateObservable.add((isRunning) => {
   }
 })
 
+function configureTaskbar(i: UnityInterface, voiceChatEnabled: boolean) {
```

Comment:
Not a big fan of this function name, but something more descriptive like`configureTaskbarRelatedHUDs` or `configureTaskbarDependentHUDs` seems too long. Can we at least add a comment there... cause it's not that clear that all those HUDs have a dependency on the Taskbar. 
Also if we can move the configuration of `USERS_AROUND_LIST_HUD` (it's the only one independent from the taskbar) outside this function, like only calling it if the `getCurrentIdentity` succeeded right below the  `configureTaskbar`, that would be great (insert meme here)